CourseApp/Pistol.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace CourseApp
+{
+    public class Pistol
+    {
+        private float caliber;
+        private int year;
+        private string model;
+        private string owner;
+        private ushort shoot = 0;
+
+        public Pistol(string model, float caliber, int year, string owner)
+        {
+            Model = model;
+            Owner = owner;
+            Caliber = caliber;
+            Year = year;
+        }
+
+        public Pistol(string model, int year, string owner)
```

Comment:
Use `this` keyword to invoke another overload of contructor
https://metanit.com/sharp/tutorial/3.1.php